### PR TITLE
Fixed includes in experimental headers

### PR DESCRIPTION
**Pull request by andriy-kokhan**

### CODE REVIEW

File: experimental/saiextensions.h [hunk 1]
```diff
@@ -25,6 +25,7 @@
 #ifndef __SAIEXTENSIONS_H_
 #define __SAIEXTENSIONS_H_
 
+#include <sai.h>
```
Comment (andriy-kokhan) [inline experimental/saiextensions.h:28]: Missing include for `SAI_API_MAX` used below

File: experimental/saiswitchextensions.h [hunk 2]
```diff
@@ -25,6 +25,7 @@
 #ifndef __SAISWITCHEXTENSIONS_H_
 #define __SAISWITCHEXTENSIONS_H_
 
+#include <saiswitch.h>
```
Comment (andriy-kokhan) [inline experimental/saiswitchextensions.h:28]: Missed include for `SAI_SWITCH_ATTR_END` used below

**Comment by kcudnik** [comment]
So this is not necessay since usually its used like
#include sai.h
#include experimental.h

**Review by kcudnik — APPROVED** [review]

**Comment by andriy-kokhan** [comment]
@rlhui , @kcudnik , can we merge this?

**Pull request merged**